### PR TITLE
[Doc] Fixed broken 'Become a Sponsor' image

### PR DESCRIPTION
**Pull request by Akshu-on-github**
**Reference issue**: #2201
- Replaced the image with the SVG ['nebulab_black.svg'](https://nebulab.com/assets/img/logo-nebulab_black.svg) on line 237
- Added alt text for images without them (lines 6, 31, 224, 233)
- <details><summary>Removed 1px SVGs found on 234 to 242</summary>On navigating to the resource links, there were SVGs that were only 1px. I figured that the code was left-over from a previously used format - but I'll add them in again if needed!</details>

I hope that all is in order, and thank you for helping me contribute to this project!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -230,20 +230,11 @@ Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com
...
-<a href="https://opencollective.com/opal/sponsor/2/website" target="_blank"><img src="https://opencollective.com/opal/sponsor/2/avatar.svg"></a>
-<a href="https://opencollective.com/opal/sponsor/3/website" target="_blank"><img src="https://opencollective.com/opal/sponsor/3/avatar.svg"></a>
-<a href="https://opencollective.com/opal/sponsor/4/website" target="_blank"><img src="https://opencollective.com/opal/sponsor/4/avatar.svg"></a>
-<a href="https://opencollective.com/opal/sponsor/5/website" target="_blank"><img src="https://opencollective.com/opal/sponsor/5/avatar.svg"></a>
-<a href="https://opencollective.com/opal/sponsor/6/website" target="_blank"><img src="https://opencollective.com/opal/sponsor/6/avatar.svg"></a>
-<a href="https://opencollective.com/opal/sponsor/7/website" target="_blank"><img src="https://opencollective.com/opal/sponsor/7/avatar.svg"></a>
-<a href="https://opencollective.com/opal/sponsor/8/website" target="_blank"><img src="https://opencollective.com/opal/sponsor/8/avatar.svg"></a>
-<a href="https://opencollective.com/opal/sponsor/9/website" target="_blank"><img src="https://opencollective.com/opal/sponsor/9/avatar.svg"></a>
```
Comment (elia) [inline README.md:242]: I think these SVGs were just for rendering potential multiple lines of backers, can you please add them back?

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
   align="center" title="Opal logo by Elia Schito" width="105" height="105" />
   <br/>
   Opal  <br/>
-  <img src="https://img.shields.io/badge/Opal-Ruby%20💛%20JavaScript-yellow.svg?logo=ruby&style=social&logoColor=777"/>
+  <img src="https://img.shields.io/badge/Opal-Ruby%20💛%20JavaScript-yellow.svg?logo=ruby&style=social&logoColor=777" alt="Opal-Ruby💛JavaScript"/>
```
Comment (elia) [inline README.md:6]: 🙏 thanks for adding all the alt attributes
nit: can you please add a space before and after the yellow heart?

**Review by elia — CHANGES_REQUESTED** [review]
Thanks!
just a couple of small things

File: README.md (hunk 1 above)
Comment (Akshu-on-github) [reply README.md]: Sure thing!

File: README.md (hunk 2 above)
Comment (Akshu-on-github) [reply README.md]: Done 😄👍

**Review by elia — APPROVED** [review]

**Pull request merged**